extensions/raycast-frc/src/frc-event.tsx

```diff
@@ -0,0 +1,255 @@
+import { useEffect, useState } from "react";
+import { format } from "date-fns";
+import { getPreferenceValues } from "@raycast/api";
+import { List } from "@raycast/api";
+import type { Award, Event, Match, Rankings } from "./frc-team";
+import { getMatchesTable } from "./frc-team";
+
+interface Arguments {
+  event: string;
+}
+
+const preferences = getPreferenceValues<{ tbaApiKey: string }>();
```

Comment:
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`